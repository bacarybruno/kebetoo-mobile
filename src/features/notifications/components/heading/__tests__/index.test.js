import setupTest from 'Kebetoo/src/config/jest-setup'

import Heading from '../index'

const givenHeading = setupTest(Heading)({
  name: 'Count',
  value: 2,
})

it('renders Heading', () => {
  const { wrapper } = givenHeading()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
