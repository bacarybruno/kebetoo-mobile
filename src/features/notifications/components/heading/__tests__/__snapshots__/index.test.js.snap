// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Heading 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginBottom": 16,
      "paddingHorizontal": 16,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "color": "#000000",
          "fontFamily": "System",
          "fontSize": 17,
          "fontWeight": "400",
          "letterSpacing": -0.408,
          "lineHeight": 22,
        },
        Array [
          Object {
            "color": "rgba(0,0,0,0.87)",
          },
          Object {
            "backgroundColor": "transparent",
            "fontFamily": "System",
            "fontWeight": "600",
          },
          undefined,
        ],
      ]
    }
  >
    Count
  </Text>
  <View
    style={
      Array [
        Object {
          "borderRadius": 1000,
          "paddingHorizontal": 16,
          "paddingVertical": 4,
        },
        Object {
          "backgroundColor": "#F0F6FE",
        },
        Object {
          "marginLeft": 8,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          false,
          false,
          Object {
            "backgroundColor": "transparent",
            "color": "#000000",
            "fontFamily": "System",
            "fontSize": 15,
            "fontWeight": "400",
            "letterSpacing": -0.24,
            "lineHeight": 20,
          },
          false,
          Array [
            Object {
              "color": "rgba(0, 122, 255, 255)",
            },
            Object {
              "backgroundColor": "transparent",
              "fontFamily": "System",
              "fontWeight": "700",
            },
            undefined,
          ],
        ]
      }
    >
      2
    </Text>
  </View>
</View>
`;
